apisix/control/router.lua

```diff
@@ -66,6 +68,38 @@ function fetch_control_api_router()
         end
     end
 
+    local discovery_type = require("apisix.core.config_local").local_conf().discovery
+    if discovery_type then
+        local discovery = require("apisix.discovery.init").discovery
+        local dump_apis = {}
+        for key, _ in pairs(discovery_type) do
```

Comment:
It seems we can use `for key, dis_mod in pairs(discovery) do` directly?

Reply:
if use `for key, dis_mod in pairs(discovery) do`, the `dis_mod` is `function` type, not a table object.

So, we can't do it.